docker-compose.yml

```
@@ -11,6 +11,7 @@ services:
     links:
       - "fossildb-persisted:fossildb"
       - "postgres-persisted:postgres"
+      - "redis:redis"
```

```suggestion
      - redis
```
This syntax is enough here, as the service-name and the dns entry match.